includes/person-functions.php

```diff
@@ -0,0 +1,20 @@
+<?php
+/**
+ * Person functions
+ */
+
```

Comment:
Do we want to use namespacing for this theme vs using function prefixes?

Reply:
For some reason I didn't start with namespacing on this project. If we want it, I can add it in throughout the project in a future PR.